src/lib/lgui/layout/simpletablelayout.cpp

```diff
@@ -250,8 +251,10 @@ MeasureResults lgui::SimpleTableLayout::measure(lgui::SizeConstraint wc, lgui::S
 }
 
 Size SimpleTableLayout::min_size_hint() {
-    int rows[mno_rows];
-    int cols[mno_cols];
+    std::vector<int> rows;
```

Comment:
Yes, I wanted to avoid a heap allocation here. But it seems to be another GNU extension (C 99 I believe).

Reply:
It wants to know the size :) Using std::vector is a rather nice hack, I think ;p